homeassistant/components/tractive/config_flow.py

```diff
@@ -69,6 +71,10 @@ async def async_step_user(
             step_id="user", data_schema=STEP_USER_DATA_SCHEMA, errors=errors
         )
 
+    async def async_step_reauth(self, _: dict = None) -> FlowResult:
+        """Prompt user to re-authenticate."""
+        return await self.async_step_user()
```

Comment:
We need to handle the flow differently on reauth. The result should be abort with reason `reauth_successful`.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#reauthentication

Note that when the integration raises ConfigEntryAuthFailed the context will by default be set with config entry id and config entry unique id, and the existing config entry data passed as user input to the reauth step, to aid the reauth flow.